modules/codegen/src/smithy4s/codegen/internals/CodegenRecord.scala

```diff
@@ -20,9 +20,11 @@ import software.amazon.smithy.model.Model
 import software.amazon.smithy.model.node.Node
 
 import scala.jdk.CollectionConverters._
+import scala.jdk.OptionConverters._
 
 private[internals] final case class CodegenRecord(
-    namespaces: List[String]
+    namespaces: List[String],
+    validatedNewtypes: Option[Boolean]
```

Comment:
If this is intended to be some configuration, this is not the right place for it. That'd be here : 

See https://github.com/disneystreaming/smithy4s/blob/89e88c972174fa0a3b2853901a52a5a6a4662c6d/modules/codegen/src/smithy4s/codegen/internals/Renderer.scala#L43-L46

Reply:
This is not meant as "configuration", but more of a record of how namespaces where previously generated.

I'm using this in the [ValidatedNewtypesTransformer](https://github.com/disneystreaming/smithy4s/pull/1454/files#diff-ecaff2416dbb6664c0ec30aa9a980b44003a627cbe092121fcad46fe481d1abeR43-R62) to make sure that namespaces that were previously generated using this new feature have the proper `@validateNewtype` trait applied.

Does that make sense?

Reply:
I see. Yeah I guess that works 👍. The approach I would have preferred would have been to generate an `apply @validatedNewType` statement in a bespoke file that would get packaged with the artifact, but I think your approach is sound too, and arguably requires less work.